atd-vzv/src/views/summary/PeopleByDemographics.js

```diff
@@ -188,7 +200,8 @@ const PeopleByDemographics = () => {
       hoverBackgroundColor: category.color,
       hoverBorderColor: category.color,
       label: category.label,
-      data: getData(category.categoryValue)
+      data: getData(category.categoryValue),
+      fullData: getData(category.categoryValue, true)
```

Comment:
Nice idea to sneak the data in this way! Very useful for the tooltip callback. 👏

Reply:
Thanks!